qemu/tests/balloon_check.py

```diff
@@ -471,6 +473,9 @@ def run(test, params, env):
     else:
         balloon_test = BallooningTestLinux(test, params, env)
 
+    if params.get("balloon_opt_deflate_on_oom") == "yes":
+        guest_ori_mem = balloon_test.get_total_mem()
+
```

Comment:
It is used by Linux only, so could we move it to `else` in L473?  just my view, it is up to you.

Reply:
@yanan-fu My bad. It can apply to windows guest too, just doesn't make any difference.  I've updated cfg, and test passed with win2016 guest. Thanks!

  (1/1) Host_RHEL.m7.u5.qcow2.virtio_scsi.up.virtio_net.Guest.Win2016.x86_64.io-github-autotest-qemu.balloon_check.balloon_evict_and_enlarge.balloon_oom: PASS (544.35 s)

Reply:
@huangyum   If so, please remember to update  polarion test case too, it be marked "rhel only" .